src/EditorFeatures/CSharpTest2/Recommendations/StackAllocKeywordRecommenderTests.cs

```diff
@@ -124,12 +90,99 @@ unsafe static void Main(string[] args)
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestInsideForStatementVarDecl3()
         {
-            await VerifyAbsenceAsync(
+            await VerifyKeywordAsync(
 @"class C
 {
     unsafe static void Main(string[] args)
     {
         for (string i = $$");
         }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestOnRHSOfAssignment_Span()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(@"
+Span<int> s = $$"));
+        }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestOnRHSOfAssignment_Var()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(
+@"var v = $$"));
+        }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestOnRHSOfAssignment_Pointer()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(
+@"int* v = $$"));
+        }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestOnRHSOfAssignment_ReAssignment()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(
+@"v = $$"));
+        }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestOnRHSWithCast()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(@"
+var s = (Span<char>)$$"));
+
+            await VerifyKeywordAsync(AddInsideMethod(@"
+s = (Span<char>)$$"));
+        }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestOnRHSWithConditionalExpression_True()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(@"
+var s = value ? $$"));
```

Comment:
hrmm.. how did this fall

Reply:
I don't understand the question?

---
In reply to: [155678159](https://github.com/dotnet/roslyn/pull/23655#discussion_r155678159) [](ancestors = 155678159)

Reply:
yeah, i don't know what this was referring to :-/